bin/gwc/db1link.ml

```diff
@@ -734,104 +747,21 @@ let update_family_with_fevents gen fam =
               in
               let () = found_marriage := true in
               loop l fam
-        | Efam_Marriage ->
-            let witnesses = Array.map fst evt.efam_witnesses in
-            let fam =
-              {
-                fam with
-                relation =
-                  (if nsck_std_fields then NoSexesCheckMarried else Married);
-                marriage = evt.efam_date;
-                marriage_place = evt.efam_place;
-                marriage_note = evt.efam_note;
-                marriage_src = evt.efam_src;
-                witnesses;
-              }
-            in
-            let () = found_marriage := true in
-            fam
-        | Efam_MarriageContract ->
-            if !found_marriage then loop l fam
-            else
-              let witnesses = Array.map fst evt.efam_witnesses in
-              (* Pour différencier le fait qu'on recopie le *)
-              (* mariage, on met une précision "vers".      *)
-              let date =
-                match Date.od_of_cdate evt.efam_date with
-                | Some (Dgreg (dmy, cal)) ->
-                    let dmy = { dmy with prec = About } in
-                    Date.cdate_of_od (Some (Dgreg (dmy, cal)))
-                | Some (Dtext _) | None -> evt.efam_date
-              in
-              (* Pour différencier le fait qu'on recopie le *)
-              (* mariage, on ne met pas de lieu.            *)
```

Comment:
do you know why we do this?

Reply:
Non justement! Je l'ai supprimé dans ma PR #1497

Reply:
let's hope it is not useful anymore!